test/LdapTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
```

Comment:
I think not only the test suite need an analysis :wink:

Will do the changes tonight!